src/test/connection_snippets.rs

```diff
@@ -0,0 +1,20 @@
+#![allow(dead_code)]
+#![allow(unused_imports)]
+#![allow(unused_variables)]
+
+use crate::error::Result;
+use crate as mongodb;
+
+#[cfg(feature = "tokio-runtime")]
+async fn connect() -> Result<()> {
```

Comment:
can we add a `#[tokio::main]` macro here and include that in the example? Otherwise users won't be able to copy/paste this snippet and have it start working. Also, does splitting up the various attributes with `///` break anything?